public/editUser.php

```diff
@@ -85,17 +82,18 @@
             $error_msg = "Password must be at least $CDASH_MINIMUM_PASSWORD_LENGTH characters.";
         }
 
-        $md5pass = md5($passwd);
-        $md5pass = pdo_real_escape_string($md5pass);
+        $password_hash = password_hash($passwd, PASSWORD_DEFAULT);
```

Comment:
I think if it fails it will be indicative of much larger problems in which case we should bail out completely, both logging the error and informing the user that something is seriously wrong. What to do with the attempted password? That I don't know.